atd-vze/src/Components/GridDateRange.js

```diff
@@ -103,7 +108,7 @@ const GridDateRange = ({ setDateRangeFilter, initStartDate, initEndDate }) => {
           startDate={startDate}
           endDate={endDate}
           minDate={startDate}
-          maxDate={maxDate}
+          maxDate={new Date()}
```

Comment:
So the max date is not persisted?

Reply:
It looks like it is persisted when I run it locally 👍

Reply:
I changed this because I noticed that the selectable date range window would narrow each time I selected a date earlier than today. This keeps the max selectable date as the current day.